llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2370,6 +2370,23 @@ struct VPFirstOrderRecurrencePHIRecipe : public VPHeaderPHIRecipe {
 #endif
 };
 
+// Possible variants of a reduction.
+
+// This reduction is ordered and in-loop.
+struct RdxOrderedInLoop {};
```

Comment:
nit:
```suggestion
struct RdxOrdered {};
```

because ordered implies in-loop.